packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -171,7 +179,7 @@ class _CupertinoScrollbarState extends RawScrollbarState<CupertinoScrollbar> {
   @override
   void updateScrollbarPainter() {
     scrollbarPainter
-      ..color = CupertinoDynamicColor.resolve(_kScrollbarColor, context)
```

Comment:
It looks like this does not currently support theming. That may be what is needed here instead.